src/js/components/user-management/selfusermanagement.js

```diff
@@ -136,6 +147,24 @@ export const SelfUserManagement = ({ canHave2FA, currentUser, editUser, hasTrack
           </p>
         </div>
       )}
+      <div className="flexbox space-between" style={{ alignItems: 'flex-start' }}>
+        <h2>Locale settings</h2>
+        <FormControl>
+          <Select
+            onChange={event => {
+              saveUserSettings({ 'locale': event.target.value });
+              console.log('Setting the user locale to: ' + event.target.value);
+              moment.locale(event.target.value);
+            }}
+            value={locale}
+          >
+            <MenuItem value={window.navigator.userLanguage || window.navigator.language}>Automatic</MenuItem>
+            <MenuItem value="en">Retard mode</MenuItem>
+            <MenuItem value="de">Deutsche telekom</MenuItem>
+            <MenuItem value="es">Spanglish</MenuItem>
```

Comment:
I'm guessing these will have to be changed, question  is just, how many to list... :thinking:

Reply:
in any case it should be a `.map` on an array of options "two or more, use a for" 😉, not sure after how many the `duplicate-code` lint rule might kick you otherwise.

Reply:
haha, fair enough. But it will be a drop-down menu of a 100+ items, is that okay?

Reply:
You can also have an array of 3 elements... `automatic, en, de, es` - or is there a requirement to cover all the formats?

Reply:
Nope :)